README.md

```diff
@@ -198,6 +197,7 @@ Create an application with two screens: Home screen and Detail screen
     - A TextField to enter the image URL.
     - Buttons to initiate image download and cancel download.
     - Display download progress using a progress bar.
+    - Add notification to show downloading progress and cancel download action
```

Comment:
Add notification to show downloading progress and cancel download action -> Add notification to show downloading progress and add cancel button in notification to cancel downloading